### ISSUE TITLE
Multiple Database Connections

### ISSUE THREAD

**Issue by sn0rcha** [mhuggins/sequelize-middleware#2]
I was wondering if there was a way to have a few instances of "models" as such available to support multiple connections. Eg:

First database connection:
`app.use(sequelize('database1', 'xxx', 'xxxx', {host: 'localhost'}, path.resolve(__dirname, 'models')));`

Second database connection:
`app.use(sequelize('database2', 'xxx', 'xxx', {host: 'localhost'}, path.resolve(__dirname, 'models_web')));`

Is it possible to have the ability to pass a couple of variables to define what the "db" and "models" variables are called?

Cheers,
Ben

**Comment by mhuggins** [comment]
That's not currently implemented, but that wouldn't be a bad idea.  Not a use case I considered. :)  To support backwards compatibility, it should support not providing this option and default to the existing naming convention, but could accept an option to override this key.

**Comment by sn0rcha** [comment]
Yep good idea, a lot of systems connect to different databases, would be great if it had this functionality!
